### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by markphillippebworth**
Misc improvements, including keeping cell type-specific calls when running combineSampleTileMatrix.

### CODE REVIEW

File: R/callOpenTiles.R [hunk 1]
```diff
@@ -620,4 +637,4 @@ setMethod(
     )
   )
```
Comment (imran-aifi) [inline R/callOpenTiles.R:638]: Save tileResults to outDir for shareMOCHA

Comment (imran-aifi) [reply R/callOpenTiles.R:791]: @markphillippebworth If we save a tileResults.rds in the directory alongside coverage files, the functions shareMOCHA (now pack/unpackMOCHA) should either only accept and return the tileResults or only the sample-tile matrix. Currently pack/unpackMOCHA can take either and will return either, which can be confusing.

Comment (markphillippebworth) [reply R/callOpenTiles.R:791]: -- we need to add a line of code to save tileResults as an HDF5 format automatically with call open tiles.
shareMOCHA should save SampleTimeMatrix as an HDF5
unpackMOCHA loads that HDF5 back in. 
loadTileResults should use a SampleTileMatrix to load the tileResults into memory from the HDF5

Comment (imran-aifi) [reply R/callOpenTiles.R:791]: Save HDF5 instead of RDS

File: R/mergeTileResults.R
Comment (imran-aifi) [inline R/mergeTileResults.R:1]: Save for another pull request

**Review by imran-aifi — CHANGES_REQUESTED** [review]

**Review by imran-aifi — APPROVED** [review]

**Pull request merged**